src/content/en/updates/2019/03/kv-storage.md

```diff
@@ -54,8 +56,15 @@ are just like regular JavaScript
 except that they don't have to be downloaded because they ship with the browser.
 
 Like traditional web APIs, built-in modules must go through a standardization
-process and have well-defined specifications, but unlike traditional web APIs,
-they're not exposed on the global scope&mdash;they're only available via
+process &mdash; each will have its own specification that requires a [design
+review](https://github.com/w3ctag/design-reviews) and positive signs of
+support from both web developers and other browser vendors before it can ship.
+(In Chrome, built-in modules will follow the same [launch
+process](https://www.chromium.org/blink/launching-features) we use to implement
+and ship all new APIs.)
```

Comment:
Suggestion: add a quick explanation about where people can see how we track "positive signs". Currently, for example on the [KV Storage ChromeStatus page](https://www.chromestatus.com/feature/6428344899862528), we have this: 

| Browser | Support |
| :---| :--- |
| Firefox: | No public signals |
| Edge: | [Public support](https://www.w3.org/2018/10/22-WebPlat-minutes.html#item05) |
| Safari: | No public signals |
| Web Developers: | Strongly positive `// 🤔 Link?` |

Reply:
@philipwalton FYI

Reply:
I don't think every new feature's article should need to explain the Blink launch process and ChromeStatus to developers. Linking, as is done here, makes sense to me as an intermediate ground.

Reply:
People were specifically critical about this piece (and may read this popular post without much context), so IMHO here it makes sense to quickly summarize the process.